### ISSUE TITLE
Could BleskOS fit on a floppy?

### ISSUE THREAD

**Issue by informer2016** [Klaykap/BleskOS#4]
This is a follow-up for https://github.com/Klaykap/LightningOS/issues/2 ;-)

**Comment by Klaykap** [comment]
Yes, compiled code is less than 50 KB, but bootloader is written to load it from USB/hard disk because nobody use floppy today. If you are interested to have some system on floppy, you can very easily rewrite bootloader to use int 13h 0x02(read from floppy) instead of 13h 0x42
